scipy/special/_support_alternative_backends.py

```diff
@@ -256,40 +299,577 @@ def fun(t, df, p):  return stdtr(df, t) - p
 
 # PyTorch doesn't implement `betainc`.
 # On torch CPU we can fall back to NumPy, but on GPU it won't work.
-_needs_betainc = xp_capabilities(cpu_only=True, exceptions=['jax.numpy', 'cupy'])
+_needs_betainc = xp_capabilities(cpu_only=True, exceptions=["jax.numpy", "cupy"])
 
 _special_funcs = (
-    _FuncInfo(_ufuncs.betainc, 3, _needs_betainc),
-    _FuncInfo(_ufuncs.betaincc, 3, _needs_betainc, generic_impl=_betaincc),
+    _FuncInfo(
+        _ufuncs.bdtr, 3,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        int_only=(False, True, False), torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.bdtrc, 3,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        int_only=(False, True, False), torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.bdtri, 3,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        int_only=(False, True, False), torch_native=False,
+    ),
+    _FuncInfo(_ufuncs.betainc, 3, _needs_betainc, torch_native=False),
+    _FuncInfo(_ufuncs.betaincc, 3, _needs_betainc, generic_impl=_betaincc,
+              torch_native=False),
+    _FuncInfo(
+        _ufuncs.betaincinv, 3,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        test_large_ints=False, positive_only=True, torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.betaln, 2,
+        xp_capabilities(cpu_only=True, exceptions=["cupy", "jax.numpy"]),
+        # For betaln, nan mismatches can occur at negative integer a or b of
+        # sufficiently large magnitude.
+        positive_only={"jax.numpy": True}, torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.binom, 2,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.boxcox, 2,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.boxcox1p, 2,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        torch_native=False,
+    ),
+    _FuncInfo(
+        _ufuncs.cbrt, 1,
+        xp_capabilities(
+            cpu_only=True, exceptions=["cupy"],
+            jax_jit=False,
+        ),
+        torch_native=False,
+    ),
     _FuncInfo(_ufuncs.chdtr, 2, generic_impl=_chdtr),
-    _FuncInfo(_ufuncs.chdtrc, 2, generic_impl=_chdtrc),
+    _FuncInfo(_ufuncs.chdtrc, 2, generic_impl=_chdtrc,
+              # gh-20972
```

Comment:
totally optional nit but just for future reference, this makes a nice hyperlink in the GitHub web UI

```suggestion
              # scipy/scipy#20972
```